packages/inferno-vnode-flags/src/index.ts

```diff
@@ -1,4 +1,5 @@
 const enum VNodeFlags {
+
```

Comment:
Known issue. Will be fixed https://github.com/prettier/prettier/issues/1480#issuecomment-299641540